sketches/BLE_Scratch/sketch.yaml

```diff
@@ -0,0 +1,16 @@
+profiles:
+  nano33ble:
```

Comment:
very good job! including profiles is a great idea.
do we need also a profile for the RP2040?

Reply:
yeah I was planning to do it in another PR